src/xcode/ENA/ENAUITests/Home/ENAUITests-Statistics.swift

```diff
@@ -97,6 +97,49 @@ class ENAUITests_Statistics: XCTestCase {
 		}
 	}
 	
+	func test_screenshot_statisticsCardTitles() throws {
+		// GIVEN
+		let title1 = AccessibilityIdentifiers.Statistics.Infections.title
+		let title2 = AccessibilityIdentifiers.Statistics.KeySubmissions.title
+		let title3 = AccessibilityIdentifiers.Statistics.Incidence.title
+		let title4 = AccessibilityIdentifiers.Statistics.ReproductionNumber.title
```

Comment:
I would prefer something more human understandable instead of title1, title1 etc.
```suggestion
		let InfectionsTitle = AccessibilityIdentifiers.Statistics.Infections.title
		let KeySubmissionsTitle = AccessibilityIdentifiers.Statistics.KeySubmissions.title
		let IncidenceTitle = AccessibilityIdentifiers.Statistics.Incidence.title
		let ReproductionNumberTitle = AccessibilityIdentifiers.Statistics.ReproductionNumber.title
```

Reply:
Fixed. I chose camel use instead.